nestjs/src/certificates/certificates.service.ts

```diff
@@ -82,4 +85,21 @@ export class CertificationsService {
       },
     };
   }
+
+  public async removeCertificate(studentId: number) {
+    const certificate = await this.certificateRepository.findOne({
+      where: { studentId },
+    });
+
+    if (!certificate) {
+      throw new NotFoundException('Certificate not found');
```

Comment:
you can do `.findOneOrFail` - you will need less code to write and error should be handled as 404